hdfs/client.py

```diff
@@ -602,6 +602,31 @@ def rename(self, hdfs_src_path, hdfs_dst_path):
         self.resolve(hdfs_src_path), hdfs_dst_path
       )
 
+  def change_owner(self, hdfs_path, owner, group=None):
+    """Change the owner of file.
+
+    :param hdfs_path: HDFS path.
+    :param owner: New owner for file.
+    :param group: Optional, new group for file.
+    """
+    if group is None:
+      res = self._set_owner(hdfs_path, owner=owner)
+    else:
+      res = self._set_owner(hdfs_path, owner=owner, group=group)
```

Comment:
Could you check that we need to handle these cases separately? (I.e. that passing `group=None` doesn't have the desired effect in the first case as well.)